scripts/mkcloud

```diff
@@ -1237,6 +1247,11 @@ function sanity_checks()
     if [[ $want_sles12 = 1 ]] && [[ $want_ceph = 1 ]] && [[ $nodenumber -lt 3 ]] ; then
         complain 113 "Please increase number of nodes for this setup, minimal nodenumber=3"
     fi
+
+    if [[ " ${steplist[*]} " == *" batch "* ]] && [[ "$scenario" == '' ]] ; then
```

Comment:
With `[[ ]]` you don't need to quote the variables.  Also `[[ -z $scenario ]]` is more idiomatic.